app/views/partners/show.html.erb

```diff
@@ -1,3 +1,4 @@
+<% @not_provided = 'Not Provided' %>
```

Comment:
I dont think we need this variable assignment anymore.

Reply:
You're absolutely right, sorry I missed that. I've fixed it.